README.md

```diff
@@ -0,0 +1,145 @@
+
+- *Follow the gradient*. Hardware-friendly **automatic differentiation** of quantum circuits.
+
+- *Batteries included*. Built-in tools for **quantum machine learning**, **optimization**, and
+  **quantum chemistry**. Rapidly prototype using built-in quantum simulators with
+  backpropagation support.
+
+## Installation
+
+PennyLane requires Python version 3.6 and above. Installation of PennyLane, as well as all
+dependencies, can be done using pip:
+
+```console
+python -m pip install pennylane
+```
+
+## Getting started
+
+For an introduction to quantum machine learning, guides and resources are available on
+PennyLane's [quantum machine learning hub](https://pennylane.ai/qml/):
+
+<img src="https://raw.githubusercontent.com/PennyLaneAI/pennylane/update-readme/doc/_static/readme/gpu_to_qpu.png" align="right" width="400px">
+
+* [What is quantum machine learning?](https://pennylane.ai/qml/whatisqml.html)
+* [QML tutorials and demos](https://pennylane.ai/qml/demonstrations.html)
+* [Frequently asked questions](https://pennylane.ai/faq.html)
+* [Key concepts of QML](https://pennylane.ai/qml/glossary.html)
+* [QML videos](https://pennylane.ai/qml/videos.html)
+
+You can also check out our [documentation](https://pennylane.readthedocs.io) for [quickstart
+guides](https://pennylane.readthedocs.io/en/stable/introduction/pennylane.html) to using PennyLane,
+and detailed developer guides on [how to write your
+own](https://pennylane.readthedocs.io/en/stable/development/plugins.html) PennyLane-compatible
+quantum device.
+
+## Tutorials and demonstrations
+
+Take a deeper dive into quantum machine learning by exploring cutting-edge algorithms on our [demonstrations
+page](https://pennylane.ai/qml/demonstrations.html).
+
+<a href="https://pennylane.ai/qml/demonstations.html">
+  <img src="https://raw.githubusercontent.com/PennyLaneAI/pennylane/update-readme/doc/_static/readme/demos.png" width="900px">
+</a>
+
+All demonstrations are fully executable, and can be downloaded as Jupyter notebooks and Python
+scripts.
+
+If you would like to contribute your own demo, see our [demo submission
+guide](https://pennylane.ai/qml/demos_submission.html).
+
+## Contributing to PennyLane
+
+We welcome contributions—simply fork the PennyLane repository, and then make a [pull
+request](https://help.github.com/articles/about-pull-requests/) containing your contribution. All
+contributers to PennyLane will be listed as authors on the releases. All users who contribute
+significantly to the code (new plugins, new functionality, etc.) will be listed on the PennyLane
+arXiv paper.
+
+We also encourage bug reports, suggestions for new features and enhancements, and even links to cool
+projects or applications built on PennyLane.
+
+See our [contributions
+page](https://github.com/PennyLaneAI/pennylane/blob/master/.github/CONTRIBUTING.md) and our
+[developers hub](https://pennylane.readthedocs.io/en/stable/development/guide.html) for more

```

Comment:
```suggestion
[developer hub](https://pennylane.readthedocs.io/en/stable/development/guide.html) for more
```
I can never decide whether it should have an apotrophe, so maybe saver without the s.